scripts/build-glean-metadata.py

```diff
@@ -75,7 +80,38 @@
             )
         )
 
-    ping_data = requests.get(PINGS_URL_TEMPLATE.format(app_name)).json()
+    def get_dependencies():
```

Comment:
Eventually we may want to move to using mozilla-schema-generator as a library, though I think it makes sense to just copy its code for now (as you are doing). However, in this context it doesn't really make sense for it to be a function, since it relies on outlying context (like app_name). Can you take out this definition and just create a `dependencies` variable that we use directly?

Can you also make a note about the original source of this function? (https://github.com/mozilla/mozilla-schema-generator/blob/master/mozilla_schema_generator/glean_ping.py)

Reply:
> Can you also make a note about the original source of this function? (https://github.com/mozilla/mozilla-schema-generator/blob/master/mozilla_schema_generator/glean_ping.py)


Have I added the comment correctly? :sweat_smile: